lib/wizard/builder.rb

```diff
@@ -160,6 +158,13 @@ def build
           value: scheme_id
         )
 
+        if scheme_id && default_theme_override && base_scheme.nil?
```

Comment:
I would add a simple comment here saying something like `# fix for the case when base_scheme is nil`